src/pages/DetailGrade/index.js

```diff
@@ -0,0 +1,264 @@
+import React, { useEffect, useState } from 'react'
+import axiosClient from 'src/axiosClient'
+import { useParams, Redirect, useHistory } from 'react-router-dom'
+import {
+  Grid,
+  Typography,
+  Container,
+  Button,
+  Box,
+  Card,
+  CardContent,
+  CardHeader,
+  IconButton,
+  CssBaseline,
+  Avatar,
+} from '@mui/material'
+import { createTheme, ThemeProvider } from '@mui/material/styles'
+import Layout from '../../Layout/Layout'
+import { DataGrid } from '@mui/x-data-grid'
+import { styled } from '@mui/styles'
+import { ArrowBackIosNew } from '@mui/icons-material'
+import { NoDataIll } from 'src/_mocks_/Illustrations'
+import accountDefault from 'src/_mocks_/account'
+import _ from 'lodash'
+import lodashGet from 'lodash/get'
+const CustomCard = styled(Card)`
+  &.sticky {
+    position: sticky;
+    top: 10px;
+    z-index: 2;
+  }
+`
+const CustomTable = styled('div')`
+  height: 400;
+  width: '100%';
+`
+const StyledDataGrid = styled(DataGrid)(({ theme }) => ({
+  border: 0,
+  color:
+    theme.palette.mode === 'light'
+      ? 'rgba(0,0,0,.85)'
+      : 'rgba(255,255,255,0.85)',
+  fontFamily: [
+    '-apple-system',
+    'BlinkMacSystemFont',
+    '"Segoe UI"',
+    'Roboto',
+    '"Helvetica Neue"',
+    'Arial',
+    'sans-serif',
+    '"Apple Color Emoji"',
+    '"Segoe UI Emoji"',
+    '"Segoe UI Symbol"',
+  ].join(','),
+  WebkitFontSmoothing: 'auto',
+  letterSpacing: 'normal',
+  '& .MuiDataGrid-columnsContainer': {
+    backgroundColor: theme.palette.mode === 'light' ? '#fafafa' : '#1d1d1d',
+  },
+  '& .MuiDataGrid-iconSeparator': {
+    display: 'none',
+  },
+  '& .MuiDataGrid-columnHeader, .MuiDataGrid-cell': {
+    borderRight: `1px solid ${
+      theme.palette.mode === 'light' ? '#f0f0f0' : '#303030'
+    }`,
+  },
+  '& .MuiDataGrid-columnsContainer, .MuiDataGrid-cell': {
+    borderBottom: `1px solid ${
+      theme.palette.mode === 'light' ? '#f0f0f0' : '#303030'
+    }`,
+  },
+  '& .MuiDataGrid-cell': {
+    color:
+      theme.palette.mode === 'light'
+        ? 'rgba(0,0,0,.85)'
+        : 'rgba(255,255,255,0.65)',
+  },
+}))
+const theme = createTheme()
```

Comment:
Theme is for app level, it use to make our source use the same config. Therefore, I think we should use only 1 theme & ThemeProvider at **app** level. Right now, I see there are 3 theme through our source code (Login.js, src/page/Profile.index.js and this file).  Could you refactor the 3 theme into main `index.js` file